.circleci/src/jobs/@frontend.yml

```diff
@@ -12,8 +12,8 @@ build-front-production:
         path: ~/fun
     - restore_cache:
         keys:
-          - v2-front-dependencies-{{ checksum "yarn.lock" }}
-          - v2-front-dependencies-
+          - v3-front-dependencies-{{ checksum "yarn.lock" }}
+          - v3-front-dependencies-
```

Comment:
:thinking: maybe the site name should be included in cache keys?

Reply:
Done